src/main/resources/application.properties

```diff
@@ -123,17 +123,19 @@ spring.mail.password:
 spring.mail.properties.mail.smtp.auth: false
 spring.mail.properties.mail.smtp.starttls.enable: false
 
-# end point for downstream and upstream search in
-# event repository
-er.url:
+# end point for downstream and upstream search in event repository
 #er.url: http://localhost:8080/eventrepository/search/
-# settings for ldap server if ldap authentication is needed
-ldap.enabled: false
-ldap.url:
-ldap.base.dn:
-ldap.username:
+er.url:
+
+# Settings for ldap server if ldap authentication is needed.
 # For security reasons and to avoid authorization problems this
-# password should be encoded. It will be decoded in EndpointSecurity.java.
-# Password needs to be encoded with base64.
-ldap.password:
-ldap.user.filter:
+# password should be encoded in base64. It will be decoded in EndpointSecurity.java.
+ldap.enabled: false
+ldap.server.list: [\
+    {\
```

Comment:
You can save one line and move up "{" to the line above -> so it becomes "{{\"

Reply:
Done.